config/proto/probe.proto

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+// Author: Evan Spendlove, GitHub: @evanSpendlove.
```

Comment:
Do we need author comments? Here and elsewhere.

Reply:
Alicja and I have discussed this with Balazs, and we're happy to go with author comments in all files.